docs/conf.py

```diff
@@ -69,7 +71,7 @@
     "sphinx_copybutton",
     "sphinx_design",
     "sphinx-prompt",
-    #"sphinxcontrib.bibtex",
+    # "sphinxcontrib.bibtex",
```

Comment:
Do we need to keep it?

Reply:
Good find. Tagging @dzier who made the original commit and likely knows more about this line. 

However, I'd like to avoid making changes wherever possible that are not related to formatting/linting in this PR given it already touches so many files.